src/gui/shareusergroupwidget.cpp

```diff
@@ -618,11 +636,27 @@ ShareUserLine::ShareUserLine(AccountPtr account,
         _permissionReshare->setVisible(false);
     }
 
+    auto avatarEventFilter = new AvatarEventFilter(_ui->avatar);
+    connect(avatarEventFilter, &AvatarEventFilter::clicked, this, &ShareUserLine::onAvatarClicked);
+    _ui->avatar->installEventFilter(avatarEventFilter);
```

Comment:
@FlexW `const auto avatarEventFilter`

Reply:
> How you would otherwise get a click event on a `QLabel`?

One alternative would be to subclass from `QLabel` and override https://doc.qt.io/qt-5.15/qlabel.html#mouseReleaseEvent. Then again, it would make this code even more complicated.

Reply:
no problem
I am not very familiar with QWidget enough to know better